fixtures/laserworld/el-400rgb-mk2.json

```diff
@@ -118,10 +118,273 @@
       }
     },
     "Color": {
-      "capability": {
-        "type": "ColorPreset",
-        "helpWanted": "What color(s) can be selected at which DMX value?"
-      }
+      "capabilities": [
+        {
+          "dmxRange": [0, 1],
+          "type": "Effect",
+          "effectName": "Color Macros",
+          "comment": "Multi Color",
+          "helpWanted": "Does this cycle all possible colours? Is this a colorFade or colorJump?"
+        },
+        {
+          "dmxRange": [2, 3],
+          "type": "ColorPreset",
+          "colors": ["#ff0000", "#00ff00", "#0000ff"],
+          "comment": "White"
+        },
+        {
+          "dmxRange": [4, 5],
+          "type": "ColorPreset",
+          "colors": ["#ff0000", "#0000ff"],
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [6, 7],
+          "type": "ColorPreset",
+          "colors": ["#00ff00", "#0000ff"],
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [8, 9],
+          "type": "ColorPreset",
+          "colors": ["#ff0000", "#00ff00"],
+          "comment": "Yellow"
+        },
+        {
+          "dmxRange": [10, 11],
+          "type": "ColorPreset",
+          "colors": ["#0000ff"],
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [12, 13],
+          "type": "ColorPreset",
+          "colors": ["#ff0000"],
+          "comment": "Red"
+        },
+        {
+          "dmxRange": [14, 15],
+          "type": "ColorPreset",
+          "colors": ["#00ff00"],
+          "comment": "Green"
+        },
+        {
+          "dmxRange": [16, 21],
+          "type": "ColorPreset",
+          "colors": ["#ff0000", "#00ff00", "#0000ff"],
+          "comment": "White"
```

Comment:
Maybe add a `helpWanted` message here why there are multiple capabilities for the same colors?